ranking/parse.py

```diff
@@ -0,0 +1,82 @@
+import re
+from helpers.common import *
+#import spdb
+#spdb.start()
+class Parser:
+    """
+    Parses tag references and tag definitions. Used for ranking
+    """
+    @staticmethod
+    def extract_member_exp(line_to_symbol,source):
+        """
+        Extract receiver object e.g. receiver.mtd() 
+        Strip away brackets and operators.
+        TODO:HIGH: Add base lang defs + Python/Ruby/C++/Java/C#/PHP overrides (should be very similar)
```

Comment:
Is this TODO still valid. What are your plans for resolving this? What happens if we use a language that doesn't have a spec included in the settings?